src/metabase/events/view_log.clj

```diff
@@ -1,10 +1,12 @@
 (ns metabase.events.view-log
```

Comment:
I'm thinking of changing this from `metabase.events.view-log` to `metabse.events.view` because this extends past the `view_log`